google-cloud-core/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Release History
 
+### 1.3.0 / 2019-01-31
+
+* Add on_error configuration (#2735)
```

Comment:
This is the only change that needs to be listed. The others should be dropped.

Please remove the PR number.